integrations/integrations.go

```diff
@@ -29,6 +29,8 @@ import (
 
 const (
 	monomerEngineWSFlag = "monomer-engine-ws"
+	defaultCacheSize    = 16 // 16 MB
+	defaultHandlesSize  = 16
```

Comment:
Any opinions on what these defaults should be?

Reply:
Seems reasonable enough.

Do we have a strategy for handling configuration?

Reply:
I reused the `RootDir` specified by the user in `svrCtx.Config`, otherwise I'd say the rest of the config is probably just going to be abstracted away to monomer. We can always revisit this if users want more control over their eth state db instance